Formula/jellyfish.rb

```diff
@@ -0,0 +1,24 @@
+class Jellyfish < Formula
+  # cite Marçais_2001: "https://doi.org/10.1093/bioinformatics/btr011"
```

Comment:
I'd avoid UTF-8 characters in formulae. Better to stick to ASCII.

Reply:
Fixed.